app/Http/Controllers/Api/ChecklistSyncController.php

```
@@ -0,0 +1,190 @@
+
+        if (! $chantierId) {
+            return response()->json(['data' => []]);
+        }
+
+        $hasAccess = Chantier::forEmployee($employee)->where('id', $chantierId)->exists();
+
+        if (! $hasAccess) {
+            return response()->json(['data' => []], 403);
+        }
+
+        $submissions = ChecklistSubmission::where('chantier_task_id', function ($q) use ($chantierId) {
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Chantiers\ChecklistSubmission::where().


(staticMethod.notFound)